libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```diff
@@ -16,7 +16,81 @@
     "overview": {
       "breadcrumb": {
         "meteringPoints": "Metering points"
+      },
+      "meteringPointType": "Type of Metering Point",
+      "meteringMethod": "Metering Method",
+      "gsrnNumber": "Metering Point ID",
+      "connectionState": "Connection State",
+      "settlementMethod": "Settlement Method",
+      "readingOccurrence": "Meter Reading Periodicity",
+      "meteringPointNotFound": {
+        "title": "Vi fandt desværre intet målepunkt [EN]",
+        "message": "Tjek for eventuelle tastefejl og prøv igen eller gå til søgning. [EN]",
+        "button": "Gå til søgning [EN]"
+      },
+      "serverError": {
+        "title": "Ups! Der opstod en fejl [EN]",
+        "message": "Der opstod desværre en fejl, da vi skulle hente de nødvendige oplysninger. [EN]",
+        "button": "Prøv igen [EN]"
```

Comment:
This one could properly just be `Retry` or `Try again` but maybe Jonas has something to say here?

Reply:
I've used `Try again` for now since that is what we use in the Danish translation.